src/main/java/com/fakng/fakngagrgtr/parser/impl/google/GoogleParser.java

```diff
@@ -0,0 +1,90 @@
+package com.fakng.fakngagrgtr.parser.impl.google;
+
+import com.fakng.fakngagrgtr.entity.Company;
+import com.fakng.fakngagrgtr.entity.Vacancy;
+import com.fakng.fakngagrgtr.parser.ApiParser;
+import com.fakng.fakngagrgtr.parser.impl.google.dto.ResponseDTO;
+import com.fakng.fakngagrgtr.parser.impl.google.dto.VacancyDTO;
+import com.fakng.fakngagrgtr.repository.CompanyRepository;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.reactive.function.client.WebClient.ResponseSpec;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class GoogleParser extends ApiParser {
+
+    private static String GOOGLE_NAME = "Google";
+    private static final DateTimeFormatter datetimeFormat =
```

Comment:
Java convention for naming constant values: DATE_TIME_FORMAT

Reply:
Done.